_includes/com-ingest/com-ingest-kafka-avro-syntax.md

```diff
@@ -0,0 +1,15 @@
+
+```json
+  "namespace": "namespace",
+  "type": "record",
+  "name": "deletes",
```

Comment:
Ok, so this include has been removed from /docs/community/com-ingest/com-ingest-source-kafka-avro.md` because as you've indicated, it's totally wrong :)

I've added the syntax directly to the file which will need reviewing :)